modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -73,6 +73,61 @@ properties:
             description: |
               Configures how long Sync Gateway should wait for a bucket operation to complete before timing out and trying again. The value can be increased in scenarios where there is a heavy load on Couchbase Server and operations are likely to take more than 2.5 seconds to complete. The default value is 2500 milliseconds.
             default: 2500
+          delta_sync:
+            type: object
+            description: |
+              *NOTE:* Delta Sync is an Enterprise Edition feature on Sync Gateway and Couchbase Lite.
+
+              Delta Sync is the ability to replicate only parts of the Couchbase mobile document that have changed. This can result in significant savings in bandwidth consumption as well as throughput improvements, especially when network bandwidth is typically constrained.
+
+              Delta Sync does not apply to attachment contents.
+
+              Delta Sync is disabled by default on the Sync Gateway. You can enable it through the `enabled` property.
+
+              If delta sync is enabled on Sync Gateway, then Couchbase Lite clients will switch to using delta sync automatically.
+              Similarly, if delta sync is disabled on Sync Gateway, clients will switch to normal mode.
+
+              The following configuration example enables delta sync.
+
+              ```javascript
+              {
+                "logging": {
+                  "console": {
+                    "log_keys": ["*"]
+                  }
+                },
+                "databases": {
+                  "db": {
+                    "server": "http://localhost:8091",
+                    "bucket": "default",
+                    "users": { "GUEST": { "disabled": false, "admin_channels": ["*"] } },
+                    "allow_conflicts": false,
+                    "revs_limit": 20,
+                    "delta_sync": {
+                      "enabled": true,
+                      "rev_max_age_seconds": 86400
+                    }
+                  }
+                }
+              }
+              ```
+
+              Foot notes:
+
+              - Delta Sync is automatically enabled for peer-to-peer sync between Couchbase Lite clients.
+              - Delta sync is disabled for Couchbase Lite database replicas.
+            properties:
+              enabled:
+                type: boolean
+                description: Set this property to "true" to enable delta sync.
+                default: 'false'
+              rev_max_age_seconds:
+                type: integer
+                description: |
+                  On write, revision body is backed up in bucket and retained for `rev_max_age_seconds`.
```

Comment:
This incurs additional bucket storage requirements, which depend on the size of documents, and the frequency of updates.

This can be estimated with the following: `(doc_size * updates_per_day * 86400) / rev_max_age_seconds`

Reply:
Good to know that the value `rev_max_age_seconds` incurs additional bucket storage requirement on CBS. Yes it's something we should document.

I will also add an example scenario:
> So with `rev_max_age_seconds`'s default value, an average doc size of 4K and 100 writes/day; enabling delta sync would take up an additional 400K on CBS (4*100*86400)/86400.

Reply:
@bbrks Done in https://github.com/couchbase/docs-sync-gateway/commit/f3c49a79ea1b201dce1b5dbbb6357d103e20f080.
I will merge this to get 2.5 out the door but I would appreciate your re-review and if it needs further amendments.